grpc-server.go

```diff
@@ -696,6 +696,17 @@ func blockContainsAccounts(block *old_faithful_grpc.BlockResponse, accounts []st
 			}
 		}
 
+		accMetaList, err := solTx.AccountMetaList()
```

Comment:
Hmm, actually I am not sure if this is giving us the accounts from accounts lookup table @gagliardetto ?